utils/__init__.py

```diff
@@ -33,3 +33,18 @@ def setup_logging(log_level=logging.DEBUG):
     logging.getLogger("urllib3").setLevel(logging.WARNING)
 
     return logger
+
+
+def time_measure(start_time, end_time, job_name=""):
+    elapsed_time = end_time - start_time
+    elapsed_minutes = elapsed_time / 60
+    log_message = f"Job {job_name}, Time: {elapsed_time:.2f}s"
+
+    if elapsed_minutes > 15:
```

Comment:
strange, why 15 minute is maximum? Just convert seconds to minutes and show 23m (for example)

Reply:
ok, make sense